### PR TITLE
T12394: Install UserProfileV2

### PR DESCRIPTION
**Pull request by songnguxyz**
Per [T12394](https://issue-tracker.miraheze.org/T12394)

## Summary by CodeRabbit

- **New Features**
	- Introduced customizable options for user profiles, including color, avatar border radius, and global avatar usage.
	- Added a new extension for enhanced user profile management with specific permissions for administrators.
- **Bug Fixes**
	- Improved clarity and organization in user profile settings with enhanced help descriptions.

### CODE REVIEW

**Comment by songnguxyz** [comment]
@coderabbitai review

File: ManageWikiSettings.php [hunk 1]
```diff
@@ -3886,6 +3895,25 @@
 		'help' => 'Whether or not to show the normal MediaWiki external link icon when using the Chameleon skin.',
 		'requires' => [],
 	],
+	'egUserProfileV2Color' => [
```
Comment (OAuthority) [inline ManageWikiSettings.php:3898]: Typo

Comment (OAuthority) [inline ManageWikiSettings.php:3898]: should be `wg`, same for below.

Comment (songnguxyz) [reply ManageWikiSettings.php:3898]: Changed, I forgot to turn the default values in `LocalSettings.php` to variables, that's why I use `eg` not `wg`

**Comment by songnguxyz** [comment]
a bit oopsies ig...

**Review by Reception123 — APPROVED** [review]

**Comment by paladox** [comment]
@Reception123 you'll probably want to create the container upv2avatars on all wikis?

**Pull request merged**